app/views/volunteers/edit.html.slim

```diff
@@ -2,7 +2,12 @@
   .col-xs-12
     h1= t_title(:edit)
 
-== render 'form'
+.row
+  .col-xs-12
+    = simple_form_for(@volunteer) do |f|
```

Comment:
PS: the form partial has .row .col-xs-12 again, so all that form rows and columns will be nested within this col-xs-12